proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,157 +44,127 @@ direction of the policies set by the Board.
-
-===========
+community. The meeting shall be directed by the CPC Chairperson.
+Responsibility for directing individual meetings may be
+delegated by the CPC Chairperson to any other CPC member. The CPC chairperson
+shall be one of the Voting members, and will be selected by the CPC members
+through consensus or if necessary a vote as described in the section titled "Voting".
+Minutes or an appropriate recording shall be taken and made available to
+the community through accessible public postings.
 
+Voting members have the roles and responsibilities as outlined below.
 
 Section 5. Responsibilities and Expectations of the CPC
 
 Subject to such policies as may be set by the Board, the CPC is responsible for
 ensuring: 
 
-   * ensuring collaboration is the driving principle within a Project, between
+   1. ensuring collaboration is the driving principle within a Project, between
      _$FOUNDATION_ Projects, and between _$FOUNDATION_ Projects and the broader
      community.
-   * defining and maintaining neutral consensus for the technical vision for the _$FOUNDATION_
-   * approving new projects within the scope for _$FOUNDATION_ set by the Governing Board,
-     as outlined in [XXXX], and creating conceptual architecture for how the proejcts
-     fit together.
-   * aligning projects, removing or archiving projects, as outlined in [XXXX].
-   * approving funds for budgets delegated to the project. A good example is the
-     travel fund.
-   * follow and be up to date on Board/$FOUNDATION initiatives and communicate them to the projects.
-   * defining common practices to be implemented across _$FOUNDATION_ projects, if any.
-   * The CPC is also responsible for mediating technical conflicts between
-     _$FOUNDATION_ Projects when attempts to resolve those conflicts within the Project
-   were unsuccessful. 
-   * The CPC serves as the _$FOUNDATION_'s primary technical
-     liaison body with external open source projects, consortiums and groups and is
-     also responsible for ensuring that the _$FOUNDATION_ has appropriate technical
-     participation in standards bodies by finding an encouraging projects members to
-     particiapte in these bodies.
-   * Members of the CPC, or committees consisting of subsets of the CPC
-     membership will be expected to meet on a regular basis to discuss topics
-     such as new project acceptance into the mentorship program, project graduation
-     from the mentorship program, Project issues and conflicts, opportunities
-     for collaboration between Projects, opportunities for the Foundation in the
-     greater JavaScript community, etc.
+   1. defining and maintaining neutral consensus for the technical vision for the _$FOUNDATION_.
+   1. creating conceptual architecture for how the projects fit together.
+   1. Voting members are responsible for approving new projects within the scope
+      for _$FOUNDATION_ set by the Governing Board, as outlined in [XXXX].
```

Comment:
is [xxxx] the process for adding projects?

if so would consulting with the board as experts for business alignment be part of that?

Reply:
Yes xxx is the the process for adding projects.  I think we should add consulting with the board to the process as opposed to in the charter.